### ISSUE TITLE
Logger priority filter does not work (for sentry)

### ISSUE THREAD

**Issue by kkrieger85** [firegento/firegento-logger#125]
May duplicate #86 .

I use two targets: Magento default files + Sentry

**Settings**

```
| Path                                | Scope   | Scope-ID | Value|
| logger/default/priority             | default | 0        | default|

| logger/general/priority             | default | 0        | 4|
| logger/general/targets              | default | 0        | default,sentry|

| logger/sentry/adminhtml_enabled     | default | 0        | 0|
| logger/sentry/curl_method           | default | 0        | async|
| logger/sentry/frontend_enabled      | default | 0        | 0|
| logger/sentry/priority              | default | 0        | 3|
| logger/sentry/public_dsn            | default | 0        | MYDSN|
```

Magento 1.9.3.10
Firegento_Logger 1.9.0

Result:
Sentry still gets notices and warnings

Expected: 
Sentry will only get errors and more severe logs

**Comment by colinmollenhour** [comment]
I'm not able to reproduce this issue.. Can you step through with a debugger and pinpoint the issue?

**Comment by colinmollenhour** [comment]
I did push a PR with some changes, but I don't know if it will fix the issue you're experiencing.
https://github.com/firegento/firegento-logger/pull/126

**Comment by wilfriedwolf** [comment]
I have got the same Problem. I found the problem:

When you look at the trace in `\mageCoreErrorHandler` you will see: 

```
Mage::log($errorMessage, Zend_Log::ERR);
```
at line 230.  This also holds for E_NOTICE and E_INFO and some other PHP Errors which should be considered notices or warnings.

Then in `\FireGento_Logger_Model_Sentry`  `$event['priority']` is always Zend_Log::ERR (3) and therefor higher as Zend_Log::NOTICE or Zend_Log::INFO as set in the priorityFilter.

This was already addressed (in `\FireGento_Logger_Model_Sentry::_assumePriorityByMessage`), but did not work.

I'll do a quick PR for this.

Best regards, 
Willi
